src/_includes/layouts/base.njk

```diff
@@ -1,28 +1,68 @@
 {% if title %}
-{% set pageTitle = title %}
+	{% set pageTitle = title %}
 {% endif %}
 <!doctype html>
-	<html lang="en-ca">
+<html lang="en-ca">
 	<head>
 		{% include 'globals/meta.njk' %}
-		<link rel="stylesheet" href="/css/main.css" />
+		<link rel="stylesheet" href="/css/main.css"/>
 		<script src="/js/main.js" defer></script>
-		{% block headerScripts %}
-		{% endblock %}
+		<link rel="preload" href="/lib/infusion/dist/infusion-uio.min.js" as="script">
+		{% block headerScripts %}{% endblock %}
+
+		<!-- CSS files for UI Options-->
+		<link rel="stylesheet" type="text/css" href="/lib/infusion/dist/assets/src/framework/preferences/css/Enactors.css"/>
+		<link rel="stylesheet" type="text/css" href="/lib/infusion/dist/assets/src/framework/preferences/css/PrefsEditor.css"/>
+		<link rel="stylesheet" type="text/css" href="/lib/infusion/dist/assets/src/framework/preferences/css/SeparatedPanelPrefsEditor.css"/>
+
+		<!-- The Infusion Library for UI Options -->
+		<script type="text/javascript" src="/lib/infusion/dist/infusion-uio.min.js"></script>
 	</head>
 	<body>
 		<div id="defaultContainer">
 			{% include 'components/header.njk' %}
 			<div class="content-wrapper">
 				<aside id="toc"></aside>
 				<main>
-				{% block content %}
-				{% endblock %}
+					<nav class="flc-toc-tocContainer"></nav>
+					{% block content %}{% endblock %}
 				</main>
 			</div>
 			{% include 'components/footer.njk' %}
 		</div>
-		{% block footerScripts %}
-		{% endblock %}
+		{% block footerScripts %}{% endblock %}
+		<script type="text/javascript">
+			$(document).ready(function () {
+				fluid.uiOptions(".flc-prefsEditor-separatedPanel", {
+					preferences: [
+						"fluid.prefs.lineSpace",
+						"fluid.prefs.textFont",
+						"fluid.prefs.contrast",
+						"fluid.prefs.tableOfContents",
+						"fluid.prefs.enhanceInputs",
+						// TODO :Simplify (not part of prefs framework for now, Justin may add it) - currently "demo.prefs.simplify"
+						// TODO: Text to speech,
+						"fluid.prefs.syllabification"
```

Comment:
Having the "syllables" panel work needs [this chunk of code](https://github.com/cindyli/infusion/blob/UIO-WeCount/demos/prefsFramework/index.html#L127-L133) too.

Let's use a separate issue ticket for adding and testing "syllables" preference. We've got enough trouble with this pull request. ;)